lib/matplotlib/axes/_axes.py

```diff
@@ -2011,6 +2011,26 @@ def step(self, x, y, *args, where='pre', data=None, **kwargs):
         kwargs['drawstyle'] = 'steps-' + where
         return self.plot(x, y, *args, data=data, **kwargs)
 
+    @staticmethod
+    def _convert_dx(x0, x, dx, convert):
```

Comment:
Not a too big deal because the API is internal, but I would move `dx` to the front:
```suggestion
    def _convert_dx(dx, x0, x, convert):
```
- `dx` is the main input to be converted, `x0` and `x` are helpers.
- Now, the first two arguments are unitful. That feels more natural than mixed unitful, unitless, unitful order